src/core/Group.h

```diff
@@ -126,7 +135,8 @@ class Group : public QObject
      * new group into another database.
      */
     Group* clone(Entry::CloneFlags entryFlags = Entry::CloneNewUuid | Entry::CloneResetTimeInfo,
-                 bool shallow = false) const;
+                 CloneFlags groupFlags = static_cast<CloneFlags>(Group::CloneNewUuid | Group::CloneResetTimeInfo |
```

Comment:
Wrong indentation

Reply:
That's actually the formatting done by `clang-format`. How do you think it should be indented?

Reply:
`CloneFlags groupFlags` should be on the same level as `Group* `

Reply:
@phoerious still not sure what you mean. `CloneFlags groupFlags`, so it should be aligned with the other parameters.

Anyway, I extracted the default flags for the function, so the signature is much smaller.

Reply:
I just got owned by the inline diff view. Ignore what I said. But not having a static_cast there is a lot cleaner, now, anyway.